tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -267,46 +277,84 @@ def test_max_connections(self):
             ],
         }
 
+        rejected_start = self._rejected_count()
+
+        def assert_no_rejected():
+            """Assert that there have been no rejected connections since the
+            start of the test."""
+            rejected_now = self._rejected_count()
+            assert rejected_now == rejected_start, f"Rejected connections detected, start={rejected_start}, now={rejected_now}"
+
         # ProducerSwarm parameters
         #
 
-        producer_kwargs = {}
-        producer_kwargs['min_record_size'] = 64
-        producer_kwargs['max_record_size'] = 64
+        record_size = 64
 
-        effective_msg_size = producer_kwargs['min_record_size'] + (
-            producer_kwargs['max_record_size'] -
-            producer_kwargs['min_record_size']) // 2
+        # very rough estimated number of connections for OMB
+        omb_connections = 3 * (omb_producer_count + omb_consumer_count)
+        # The remainder of our connection budget after OMB connections are accounted for we will
+        # fill with swarm connections: we add 10% to the nominal amount to ensure we test the
+        # advertised limit and this uses up ~half the slack we have in the enforcement (we currently
+        # set the per broker limit to 1.2x of what it would be if enforced exactly).
+        swarm_target_connections = int(
+            (tier_limits.max_connection_count - omb_connections) * 1.1)
 
-        conn_limit = tier_limits.max_connection_count - 3 * (total_producers +
-                                                             total_consumers)
-        _target_per_node = conn_limit // SWARM_WORKERS
-        _conn_per_node = int(_target_per_node * 0.7)
+        # we expect each swarm producer to create 1 connection per broker, plus 1 additional connection
+        # for metadata
+        conn_per_swarm_producer = self.num_brokers + 1
 
-        msg_rate_per_node = (1 * KiB) // effective_msg_size
-        messages_per_sec_per_producer = max(
-            msg_rate_per_node // _conn_per_node, 1)
+        producer_per_swarm_node: int = swarm_target_connections // conn_per_swarm_producer // SWARM_WORKERS
+
+        messages_per_sec_per_producer = 1
+        msg_rate_per_node = messages_per_sec_per_producer * producer_per_swarm_node
 
         # single producer runtime
-        # Roughly every 500 connection needs 60 seconds to ramp up
-        time_per_500_s = 60
-        warm_up_time_s = max(
-            time_per_500_s * math.ceil(_target_per_node / 500), time_per_500_s)
-        target_runtime_s = 60 * (test_duration +
+        # Each swarm will throttle the client creation rate to about 30 connections/second
+        warm_up_time_s = producer_per_swarm_node // 30 + 60
```

Comment:
Should this just use DEFAULT_CLIENT_SPAWN_WAIT_MS?

Reply:
Yes it should (I added the constant latter and didn't go back and fix this usage).

Reply:
This is fixed in https://github.com/redpanda-data/redpanda/compare/5d9ad58e47691dbc16c82a9e1b5b413cab4307e0..837694d18838e6329dab858c39c2e930e831ccfc